### ISSUE TITLE
Issues playing podcasts with screen readers for the visually impaired

### ISSUE THREAD

**Issue by amirsol81** [hello-efficiency-inc/raven-reader#478]
**Describe the bug**
1. The buttons in the Podcast Player have no labels, meaning we can't tell, via screen readers for the blind such as NVDA and JAWS, what they do unless we select them to learn their functionality. Screen readers just report them as "Button".
2. Don't know if Raven Reader supports this, but it is not possible to move backwards and forwards in the currently playing podcast. Screen readers just report the elapsed/collapsed time, but can't interact with the Playback control to alter the so-called cursor in the podcast being played.
**To Reproduce**
I suggest that in order to duplicate this and other issues mentioned in #475 the open-source NVDA screen reader be installed from here: https://www.nvaccess.org/download/
**Expected behavior**
Buttons in the Podcast Player should have descriptive labels, and if the app supports it, screen readers should be allowed to alter the playback position.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Win 10 Pro 64-Bit, Version 2004
 - Browser Google Chrome 64-Bit
 - Version 85

**Smartphone (please complete the following information):**

**Additional context**
Using the latest app release from Raven Reader's official website.

**Comment by mrgodhani** [comment]
Made the podcast player accessible in the latest release https://ravenreader.app/download/ 1.0.3

**Comment by amirsol81** [comment]
Thanks for looking into this.
